lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_build_to_apps_cdn.rb

```diff
@@ -34,8 +34,10 @@ class UploadBuildToAppsCdnAction < Action
         'Mac - Silicon',
         'Mac - Intel',
         'Mac - Any',
-        'Windows',
-        'Microsoft Store',
+        'Windows - x86',
+        'Windows - ARM64',
+        'Microsoft Store - x86',
+        'Microsoft Store - ARM64',
```

Comment:
ℹ️  As discussed in https://linear.app/a8c/issue/AINFRA-1457/update-appscdn-backend-and-release-toolkit-for-new-windows#comment-646bbf97, we've agreed to get rid of the original plain `'Windows'` and `'Microsoft Store'` values (as opposed to keeping them for backwards compatibility) since Studio was the only client product to use those old non-architecture-specific values in the first place.